include/mbedtls/config_psa.h

```diff
@@ -355,6 +355,90 @@ extern "C" {
 #endif /* PSA_WANT_KEY_TYPE_CHACHA20 */
 #endif /* PSA_WANT_ALG_CHACHA20_POLY1305 */
 
+#if defined(PSA_WANT_ECC_SECP192R1)
```

Comment:
```suggestion
#if defined(PSA_WANT_ECC_SECP_R1_192)
```
See https://github.com/ARMmbed/mbedtls/blob/2c5d9e6a32ec2e131eaf1c015dab65b2ff83059e/docs/proposed/psa-conditional-inclusion-c.md (the document is already in the tree but https://github.com/ARMmbed/mbedtls/pull/4133 clarifies that section).

The macro names have this structure so that they can be derived mechanically from the `PSA_ECC_FAMILY_xxx` symbol and the key size.